### ISSUE TITLE
Add support for custom tailwind configuration

### ISSUE THREAD

**Issue by sebszocinski** [vinkla/autocomplete-tailwind#9]
It would be nice if the autocomplete could only suggest from your configured elements for the current site.. so it'd have to read the `tailwind.js` file and populate from that?

**Comment by vinkla** [comment]
I agree it would be a great feature to include in this package. I actually tried building an updated version which generated the suggestions based on the user configuration. Though, I didn't manage to solve it without making the package slow on activation.

**Comment by sebszocinski** [comment]
Thanks for the quick response @vinkla and yes that would be a great feature!

**Comment by sebszocinski** [comment]
@vinkla do you mind sharing what you made so far?

**Comment by vinkla** [comment]
I've published the `custom-config` branch which I worked on before Tailwind 1.0, [you can find it here](https://github.com/vinkla/autocomplete-tailwind/commit/d2b12c1b5dc8bf3b5714c4e972cbf9183d203a94). Since then I've rewritten the [`build`](https://github.com/vinkla/autocomplete-tailwind/blob/e6fbdddc22ee097faf95b65b571ef5e2737f6008/script/build.js) script. We no longer use the `tailwind-class-names` package since it doesn't seem to support later versions of Tailwind.

**Comment by sebszocinski** [comment]
Ah ok so this won't work with Tailwind v1.0+ ?

**Comment by vinkla** [comment]
Not in its current state no.

**Comment by sebszocinski** [comment]
hey @vinkla any updates on this?

**Comment by vinkla** [comment]
Don’t have the time to implement this feature at the moment. 

You’re welcome to send a pull request!

**Comment by desaintflorent** [comment]
Like @sebszocinski  I'm also very interested for custom tailwind configuration integrated with your plugin ! 
Can't do any PR myself unfortunately :/
